src/Codeception/Lib/Generator/Actions.php

```diff
@@ -255,10 +260,24 @@ private function createReturnTypeHint(ReflectionMethod $refMethod): string
             return '';
         }
 
+        return ': ' . $this->stringifyType($returnType);
+    }
+
+    /**
+     * @param \ReflectionType $returnType
+     * @return string
+     */
+    private function stringifyType(\ReflectionType $returnType)
```

Comment:
I think you can `: string` and remove that docblock.

Reply:
Done